tests/harvest/test_scenarios.py

```
@@ -159,19 +160,29 @@ def test_scenario_one(
     store_info_sets,
+    create_harvest_service,
 ):
     assert httpd_server_base
     for store in store_info_sets:
-        log.debug(f"testing scenario one for {store}")
-        config = CONFIGS / "dereference_test2_sparql.yml"
-        travharv = service(
-            config,
-            store,
-        )
+        log.debug(f"testing scenario two for {store}")
```

Ruff

Logging statement uses f-string

(G004)